lnwallet/chancloser/chancloser.go

```diff
@@ -837,6 +873,175 @@ func (c *ChanCloser) proposeCloseSigned(fee btcutil.Amount) (*lnwire.ClosingSign
 	return closeSignedMsg, nil
 }
 
+// handleRemoteProposal sanity checks the remote's ClosingSigned message and
+// tries to determine an acceptable fee to reply with. It may return:
+// - a message and no error (continue negotiation)
+// - no message and an error (negotiation failed)
+// - no message and no error (indicating we agree with the remote's fee)
+func (c *ChanCloser) handleRemoteProposal(
+	remoteMsg *lnwire.ClosingSigned) (lnwire.Message, error) {
+
+	isFunder := c.cfg.Channel.IsInitiator()
+
+	// If FeeRange is set, perform FeeRange-specific checks.
+	if remoteMsg.FeeRange != nil {
```

Comment:
I thought fee range was actually just in _shutdown_?

Reply:
it's in closing_signed